.github/workflows/tests.yml

```diff
@@ -158,6 +158,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo test --no-run --target=aarch64-apple-darwin --features=std
       - run: cargo test --no-run --target=aarch64-apple-ios --features=std
+      - run: cargo test --no-run --target=aarch64-apple-watchos-sim -Zbuild-std --features=std
```

Comment:
You need to update the `dtolnay/rust-toolchain@stable` to `dtolnay/rust-toolchain@nightly` and add `components: rust-src` to the `with:`, so something like:
```yaml
      - uses: dtolnay/rust-toolchain@nightly
        with:
          targets: aarch64-apple-darwin, aarch64-apple-ios
          components: rust-src
```